examples/DrawXNative/DrawX/MyClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace DrawX
+{
+    public class MyClass
```

Comment:
Lets add an issue to _finish_ the DrawX forms sample, or decide to drop it entirely. @kristiandupont @fealebenpae do you have a preference?